plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Privileges.js

```diff
@@ -208,13 +209,14 @@ Ext.define('NX.coreui.controller.Privileges', {
    */
   deleteModel: function (model) {
     var me = this,
-        description = me.getDescription(model);
+        name = model.get('name');
 
     NX.direct.coreui_Privilege.remove(model.getId(), function (response) {
       me.loadStore();
       if (Ext.isObject(response) && response.success) {
         NX.Messages.add({
-          text: 'Privilege deleted: ' + description, type: 'success'
+          text: NX.I18n.format('ADMIN_PRIVILEGES_MESSAGE_DELETED', name),
```

Comment:
minor but given it's single use we could ditch the temp variable and just call it here?